home/make-user.js

```diff
@@ -0,0 +1,19 @@
+function makeUser(formData) {
+    const user = {
+        name: formData.get('name'),
+        madness: 100,
+        bank: '$10',
+        completed: {}
+    };
+
+    return user;
+}
+
+export const newMakeUser = (formData) => ({
+    name: formData.get('name'),
+    madness: 100,
+    bank: '$10',
+    completed: {}
+});
```

Comment:
This function does the same thing as the one above and isn't being imported anywhere so you can probably get rid of it.